packages/components/src/card/card/hook.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import deprecated from '@wordpress/deprecated';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../../ui/context';
+import { useSurface } from '../../ui/surface';
+import * as styles from '../styles';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').Props, 'div'>} props
+ */
+function useDeprecatedProps( { elevation, isElevated, ...otherProps } ) {
+	let computedElevation = elevation;
+
+	if ( isElevated ) {
+		deprecated( 'Card isElevated prop', {
+			since: '5.8',
```

Comment:
I think it would actually be `5.9`, right? `5.8` is already going to be released by the time this merges.

Reply:
Yeah, that's true. Updated in https://github.com/WordPress/gutenberg/pull/32243/commits/7f624d792f0b7f31f23b71e51362d83364e68309